evalml/pipelines/time_series_regression_pipeline.py

```diff
@@ -209,15 +209,32 @@ def get_prediction_intervals(
             X_train=X_train,
             y_train=y_train,
         )
-        if self.estimator.model_family in self.NO_PREDS_PI_ESTIMATORS:
+        has_stl = "STL Decomposer" in list(
```

Comment:
We're probably going to want to make this slightly more robust here and check for `STLDecomposer.name` instead of hardcoding the decomposer name. The same holds below on line 229 as well.